infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -72,12 +71,13 @@
  * . When the entry is complete, a written flag is set in the flags field.
  */
 @Slf4j
+@Builder
 public class LogUnitServer extends AbstractServer {
 
     /**
      * The options map.
      */
-    private final LogUnitServerConfig config;
+    private final LogUnitParameters config;
```

Comment:
Technically when this object is constructed it is considered as arguments and not parameters.
Also, arguments/parameters might not reflect this objects purpose precisely because this object takes some arguments and transforms them into configurations (i.e. the conversion of logSizeLimitPercentage to logSizeLimit which I think should belong inside LogUnitConfig).